source/slides/10_cfg_mgt.rst

```diff
@@ -2,3 +2,74 @@
 
 Configuration Management History & Basics
 =========================================
+
+
+DevOps workflow in an agency environment
+========================================
+
+Squishymedia
+
+.. image:: /_static/squishy.png
+
+Greg Lund-Chaix
+Director of Technology
+
+
+Tools @ Squishy
+===============
+
+* GitlLab & GitLab CI - http://gitlab.com
+* Puppet - http://puppetlabs.com
+* Vagrant - http://vagrantup.com
+
+Workflow @ Squishy
+==================
+
+Repository layout:
+
```

Comment:
We have some css to make large code blocks smaller. If you add ``.. rst-class:: codeblock-sm`` here that will help. But it still looks like it cuts off most of this on the slide. You may need to split this into two slides if you want to show the whole thing.